runtime/frequency/src/lib.rs

```diff
@@ -112,6 +113,14 @@ impl ProposalProvider<AccountId, RuntimeCall> for CouncilProposalProvider {
 	}
 }
 
+pub struct CapacityBatchProvider;
+
+impl UtilityProvider<RuntimeOrigin, RuntimeCall> for CapacityBatchProvider {
+	fn batch_all(origin: RuntimeOrigin, calls: Vec<RuntimeCall>) -> DispatchResultWithPostInfo {
```

Comment:
We retained `batch_all` in the naming because this function is serving as a passthrough, wrapping the utility pallet's `batch_all` extrinsic function. I _think_ their rationale is that `pallet_utility::batch` is not atomic whereas `pallet_utility::batch_all` is